<launch>
    <node name="segment_pc_node" pkg="segpc_yolov7" type="segment_pc_node" />
    <rosparam file="$(find segpc_yolov7)/config/params.yaml" />
</launch>